templates/ts/sso-tab-with-obo-flow/package.json.tpl

```diff
@@ -2,7 +2,7 @@
   "name": "{{SafeProjectNameLowerCase}}",
   "version": "0.1.0",
   "engines": {
-    "node": "16 || 18"
+    "node": "18 || 20"
```

Comment:
Do we need to update the @types/node dependency version?

Reply:
I tested it on my local machine and it worked without any errors or warnings in the editor. If we had used any features from Node 20, we should have upgraded this dependency, but we didn't.